tests/scoring_engine/web/test_api.py

```diff
@@ -0,0 +1,28 @@
+from web_test import WebTest
+
+
+class TestAdmin(WebTest):
+
+    def test_auth_required_api_admin_get_round_progress(self):
+        self.verify_auth_required('/api/admin/get_round_progress')
+
+    def test_auth_required_api_admin_get_teams(self):
+        self.verify_auth_required('/api/admin/get_teams')
+
+    def test_auth_required_api_admin_update_password(self):
+        resp = self.client.post('/api/admin/update_password')
```

Comment:
Instead of having these 3 lines, we can simply call "self.verify_auth_required('/api/admin/update_password')" which does that work for us.